### PR TITLE
Amoeba_net_d on tpupod

### PR DESCRIPTION
**Pull request by bignamehyp**
Hello,

Similar to PR #18, we trained amoeba_net_d on portions of a TPUv2 pod.

Please let me know if you have any questions!

### CODE REVIEW

**Comment by bignamehyp** [comment]
R: @codyaustun

Thank you very much!

File: google_amoeba_net_d_quater_tpupod_tensorflow18.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "framework": "TensorFlow 1.8.0-rc1",
+  "codeURL": "https://github.com/tensorflow/tpu/tree/e5c126d66aa3d25e0cb066bdf7fc46f98fe59901/models/experimental/amoeba_net/",
+  "model": "AmoebaNet-D N6F256",
+  "hardware": "1/4 of a TPUv2 Pod",
+  "timestamp": "2018-04-20",
+  "misc": {
+    "optimizer": "RMSProp",
+    "movingAverageDecay": 0.9999,
+    "weightDecay": 0.00004,
+    "batchSize": 4096,
```
Comment (codyaustun) [inline google_amoeba_net_d_quater_tpupod_tensorflow18.json:14]: Should the batch size for 1/4 of TPUv2 Pod be the same as the batch size for 1/16 of a TPU Pod? Looking at the [previous AmoebaNet-D submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/77f9503ad2a4ee96e652efcef1cb930f6887d730/ImageNet/train/google_amoeba_net_d_tpu_tensorflow18.json#L15) you used a batch size of 1024. I imagine you would have to scale the batch size as the hardware increases, but I don't see that here between 1/16 to 1/4.

File: google_amoeba_net_d_quater_tpupod_tensorflow18.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "warmUp": "the learning rate starts at 0, then it increases linearly per iteration for the first 3 epochs",
+    "exponentialDecay": "The initial learning rate is 10.24, it decrease by a factor of 0.88 every one epoch",
+    "numEpochs": 41,
```
Comment (codyaustun) [inline google_amoeba_net_d_quater_tpupod_tensorflow18.json:17]: Why did the number of epochs increase here? Previously you used 35 epochs for the Cloud TPU and 1/16 of a TPUv2 Pod.

File: google_amoeba_net_d_quater_tpupod_tensorflow18.json (hunk 1 above)
Comment (bignamehyp) [reply google_amoeba_net_d_quater_tpupod_tensorflow18.json:14]: Nope, for both 1/16 and 1/4 tpu pod the total batch size is 4096.

File: google_amoeba_net_d_quater_tpupod_tensorflow18.json (hunk 2 above)
Comment (bignamehyp) [reply google_amoeba_net_d_quater_tpupod_tensorflow18.json:17]: To ensure reproducibility, we repeated our runs multiple times and summit the bottom 30% result to DAWN.  It won't train to 93% in 35 epochs for most runs. There are more TPU cores in a 1/4 TPUv2 pod but the total batch size is the same. The smaller per-core batch size might affect the batch normalization so it takes more epochs to train to 93%.

File: google_amoeba_net_d_quater_tpupod_tensorflow18.json (hunk 1 above)
Comment (bignamehyp) [reply google_amoeba_net_d_quater_tpupod_tensorflow18.json:14]: In general, it requires a lot of tuning to train with a very large batch size while achieving the same accuracy. Sometimes it's even impossible. For example, the largest batch size to train ResNet to 76%+ accuracy is 8k.  See table 1 in
Ref: https://arxiv.org/pdf/1711.04325.pdf

Comment (codyaustun) [reply google_amoeba_net_d_quater_tpupod_tensorflow18.json:14]: Thanks for confirming that this was intentional. I am surprised that the maximum batch size is lower for this architecture in comparison to ResNet.

File: google_amoeba_net_d_quater_tpupod_tensorflow18.json (hunk 2 above)
Comment (codyaustun) [reply google_amoeba_net_d_quater_tpupod_tensorflow18.json:17]: Thanks for clarifying.

**Comment by codyaustun** [comment]
@bignamehyp Please submit a new pull request for this. You didn't put these files in the ImageNet/train directory. The website won't display the results until this is corrected.

**Comment by bignamehyp** [comment]
@codyaustun 
Sorry about that. Created #37
Please take a look
